UserApi/Testing.Common/ActiveDirectory/ActiveDirectoryUser.cs

```diff
@@ -9,56 +9,51 @@ namespace Testing.Common.ActiveDirectory
 {
     public static class ActiveDirectoryUser
     {
+        private static string ApiBaseUrl => $"https://graph.microsoft.com/v1.0/{TestConfig.Instance.AzureAd.TenantId}";
 
         public static async Task<bool> IsUserInAGroup(string user, string groupName, string token)
         {
-            using (var client = new HttpClient())
-            {
-                client.DefaultRequestHeaders.Authorization =
-                    new AuthenticationHeaderValue("Bearer", token);
-                var httpRequestMessage = new HttpRequestMessage(HttpMethod.Get,
-                    $@"https://graph.microsoft.com/v1.0/users/{user}/memberOf");
-                var result = client.SendAsync(httpRequestMessage).Result;
-                var content = await result.Content.ReadAsStringAsync();
-                return content.Contains(groupName);
-            }
+            var url = $@"{ApiBaseUrl}/users/{user}/memberOf";
+            var groupsResult = await SendGraphApiRequest(url, HttpMethod.Get, token);
```

Comment:
Maybe Move param HttpMethod as first param, so I can read it as 
GET -> URL -> with Token
Just my preference,  you don't really need to do it

Reply:
No, that makes sense. I did feel something was a bit off with it when I wrote it :)